### PR TITLE
german translation, example project optimized, admin style

### PR DESCRIPTION
**Pull request by zypro**
This is an old fork... but there should not be any "conflicts".
- german translations
- extrahead style removed for other packages like django-cms
- updated example project and other repo settings

### CODE REVIEW

File: docs/advanced_options.rst [hunk 1]
```diff
@@ -275,3 +275,6 @@ Settings
 
 ``MODERATION_MODERATORS``
     Tuple of moderators' email addresses to which notifications will be sent.
+
+``DEFAULT_FROM_EMAIL``
```
Comment (DmytroLitvinov) [inline docs/advanced_options.rst]: This settings is not related to this library so maybe we do not need it here?

Comment (dominno) [reply docs/advanced_options.rst]: exactly, this is default Django setting, no need for it here

File: moderation/templates/moderation/moderate_object.html [hunk 2]
```diff
@@ -1,43 +1,5 @@
 {% extends "admin/change_form.html" %}
 {% load i18n %}
-{% block extrahead %}
-<style type="text/css">
```
Comment (dominno) [inline moderation/templates/moderation/moderate_object.html]: Why are you removing this ? as far as I remember this was need to display properly the image fields. Does it work on image fields without it ?

Comment (zypro) [reply moderation/templates/moderation/moderate_object.html]: Okay, I'll undo this, but add a {{ block.super }} to this section to get other extraheads (e.g. from django-cms).

File: docs/advanced_options.rst (hunk 1 above)
Comment (zypro) [reply docs/advanced_options.rst]: removed it

File: moderation/migrations/0005_auto_20190412_0442.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.18 on 2019-04-12 09:42
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```
Comment (dominno) [inline moderation/migrations/0005_auto_20190412_0442.py]: What is this migration for ? I dont see any changes in models file. Why do you need this migrations ?

Comment (zypro) [reply moderation/migrations/0005_auto_20190412_0442.py]: They are auto-generated by django... don't understand why. There are no changes in the `models.py`... Did you forget to commit them earlier?

Comment (dominno) [reply moderation/migrations/0005_auto_20190412_0442.py]: ok I see.
